webrender/src/frame.rs

```diff
@@ -548,8 +535,7 @@ impl Frame {
                                       &iframe_stacking_context,
                                       iframe_clip);
 
-        context.builder.pop_clip_scroll_node();
-        context.builder.pop_clip_scroll_node();
```

Comment:
we still push the clip scroll nodes (https://github.com/servo/webrender/pull/978/files#diff-054157e919dfed0477381da46476b556L531). Where do we pop them now?